l1-contracts/test/decoders/Base.sol

```diff
@@ -6,6 +6,7 @@ import {Test} from "forge-std/Test.sol";
 
 contract DecoderBase is Test {
   // When I had data and messages as one combined struct it failed, but I can have this top-layer and it works :shrug:
+  // Note: Members of the struct (and substructs) have to be in ALPHABETICAL order!
```

Comment:
Thanks for adding the extra clarity here in the note. Think I just mentioned it 😬